mono/profiler/log.c

```diff
@@ -271,6 +269,7 @@ static MonoLinkedListSet profiler_thread_list;
  * 	[name: string] image file name
  * if mtype == TYPE_ASSEMBLY
  * 	[name: string] assembly name
+ * 	[image name: string] image name
```

Comment:
This doesn't map well with the runtime difference between Assembly and Image.
It could lead to weird semantics in the future.
Couldn't we add a Image* instead?